internal/ingress/controller/controller.go

```diff
@@ -1069,6 +1071,9 @@ func (n *NGINXController) createServers(data []*extensions.Ingress,
 				if err != nil {
 					glog.Warningf("SSL certificate %q does not contain a Common Name or Subject Alternative Name for server %q: %v",
 						secrKey, host, err)
+					glog.Warningf("Using default certificate")
```

Comment:
Doesn't make too much sense to me after this PR, because the default cert is not likely to contain a valid SAN for that domain either. Maybe we should let the user set that certificate anyway and just print the warning?